cost/azure/hybrid_use_benefit_sql/ahub_sql.pt

```diff
@@ -241,6 +241,8 @@ script "js_filtered_instances", type: "javascript" do
   result "result"
   code <<-EOS
   var result = [];
+  sql_virtualmachines = _.filter(sql_virtualmachines, function(vm){ return vm.imageSku != "Developer" && vm.imageSku != "Express" });
```

Comment:
```suggestion
  // Filter to exclude SQL Server 2017 Developer and Express versions instances that Microsoft provides as free
  sql_virtualmachines = _.filter(sql_virtualmachines, function(vm){ return vm.imageSku != "Developer" && vm.imageSku != "Express" });
```
Suggest commenting the code so next person knows what this filter does in the code